spring-cloud-skipper-server-core/src/main/resources/db-schema/schema-db2.sql

```diff
@@ -176,7 +177,6 @@
 
     create table skipper_status (
         id bigint not null,
-        object_version bigint,
```

Comment:
I guess `object_version` needs to be removed in some other places where they are still being used?

Reply:
This is due to changed super class in commit https://github.com/spring-cloud/spring-cloud-skipper/commit/53b83cc0756d10517caca4fadd24807608d71003